src/components/CalendarToolbar.tsx

```diff
@@ -0,0 +1,116 @@
+"use client";
```

Comment:
Si CalendarToolbar est utilisé dans Calendar, et que celui ci a déjà un "useClient" tu n'as pas besoin de le spécifier à nouveau ici (idem pour les autres components BCalendar).

https://nextjs.org/docs/app/building-your-application/rendering#network-boundary

Reply:
got it